.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Update style.

Reply:
????

Reply:
I removed the qoutes from the object props that do not need it. is that what you meant?